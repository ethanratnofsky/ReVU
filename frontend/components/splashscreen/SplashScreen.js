import { Image, Text, View } from "react-native";

import splashScreenStyles from "./SplashScreenStyles";

const SplashScreen = () => {
    return (
        <View style={splashScreenStyles.container}>
            <Image
                style={splashScreenStyles.logo}
                source={require("../../assets/images/revu-logo.png")}
            />
            <View style={splashScreenStyles.credits}>
                <Text style={splashScreenStyles.creditLine1}>from</Text>
                <Text style={splashScreenStyles.creditLine2}>Vanderbilt Student Government</Text>
            </View>
        </View>
    );
};

export default SplashScreen;
