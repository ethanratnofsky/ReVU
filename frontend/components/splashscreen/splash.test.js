import React from 'react';
const renderer = require('react-test-renderer');
import SplashScreen from './SplashScreen';

describe('<SplashScreen />', () => {
  it('has 2 children', () => {
    const tree = renderer.create(<SplashScreen />).toJSON();
    expect(tree.children.length).toBe(2);
  });
});

it('renders correctly', () => {
    const tree = renderer.create(<SplashScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });