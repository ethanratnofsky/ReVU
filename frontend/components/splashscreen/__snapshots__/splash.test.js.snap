// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1C1C1C",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 50,
        "width": 172,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 35,
        "position": "absolute",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 12,
          "fontWeight": "bold",
        }
      }
    >
      from
    </Text>
    <Text
      style={
        Object {
          "color": "#CFAE70",
          "fontSize": 14,
          "fontWeight": "bold",
        }
      }
    >
      Vanderbilt Student Government
    </Text>
  </View>
</View>
`;
